data/dev.geopjr.Collision.json

```diff
@@ -61,7 +61,7 @@
         {
           "type": "git",
           "url": "https://gitlab.gnome.org/GNOME/libadwaita.git",
-          "tag": "1.0.2"
+          "tag": "1.1.0"
```

Comment:
Libadwaita 1.1.0 is in the master SDK, is it necessary to explicitly specify the use of libadwaita as a project module? If not, sassc and libsass modules could also be removed.

Reply:
Wasn't aware, fixed, thanks!